### DISCUSSION TITLE
More User friendly UI/UX Startup Screen

### DISCUSSION THREAD

**Discussion by ghost** [4rkal/moneroOS#7] [Ideas]
Just an issue for others to put their suggestions here:

**Comment by 4rkal** [comment]
You can understand that since MoneroOS is still in early development UI/UX is not the first priority. I will be updating the ui in the future but right now were not at that point. 
I will however take any suggestions and accept any pull requests that come my way (if they meet certain criteria). The current UI implementation can be found here: https://github.com/4rkal/MoneroOS/blob/master/airootfs/root/bar.sh. Also note that there is no plan to change from bashsimplecurses unless something more performant is found.

**Comment by ghost** [comment]
> You can understand that since MoneroOS is still in early development UI/UX is not the first priority. I will be updating the ui in the future but right now were not at that point. I will however take any suggestions and accept any pull requests that come my way (if they meet certain criteria). The current UI implementation can be found here: https://github.com/4rkal/MoneroOS/blob/master/airootfs/root/bar.sh. Also note that there is no plan to change from bashsimplecurses unless something more performant is found.

Thanks, this issue can be moved to discussions
